.github/mentor-list.template.md

```diff
@@ -0,0 +1,44 @@
+# Template para a lista de mentores(as)
+
+Adicione seu nome ao próximo a outra pessoa que presta mentoria relacionado ao assunto que você deseja contribuir.
+
+Por exemplo:
+
+| NOME | TIPO DE MENTORIA | CONTATO
+| --- | --- | --- |
+| William Oliveira | Front-End e Nodejs | @meutelegram
+
+Se você for mentorar sobre front-end, então adicione seu nome próximo a última pessoa que se cadastrou como front-end, nesse caso seria o William Oliveira.
```

Comment:
Mesma coisa aqui. "Se você for mentorar sobre front-end, então adicione seu nome _**logo após**_ o da última pessoa que se cadastrou como front-end, nesse caso seria o William Oliveira"

Reply:
Show de bola! Vou alterar.